misc/device_module_config.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
how/where is this file used?  I don't see it referenced anywhere. Also, I think we're getting the point where we should likely isolate this stuff in a separate "qualification" directory to keep misc/ from getting polluted.  So, maybe move these under "daq/qualification" to provide the room for future stuffs!

Reply:
Sounds like a good plan. I'll create the new qualification folder.